3-platform/order/boot/src/main/resources/application.properties

```
@@ -1,3 +1,7 @@
 application.banner.title=Order Service
 application.banner.version=@project.version@
 spring.output.ansi.enabled=ALWAYS
+
+spring.r2dbc.url=r2dbc:postgresql://localhost:5432/order_service_db

```

for time being lets have database_name=postgres
when we will do deployment, we change it to proper, but for now in all services `postgres`.
```suggestion
spring.r2dbc.url=r2dbc:postgresql://localhost:5432/postgres

```